lms/templates/ccx/grading_policy.html

```diff
@@ -2,7 +2,7 @@
 
 <div id="warn-coach" class="wrapper-msg urgency-high warning">
   <div class="msg">
-    <i class="msg-icon icon-warning-sign"></i>
+    <i class="msg-icon fa fa-warning"></i>
```

Comment:
This looks like Font-Awesome, so I changed this, although this CCX stuff is not something I've worked in before...